lib/src/otx/data/samplers/class_incremental_sampler.py

```diff
@@ -65,7 +64,7 @@ def __init__(
         super().__init__(dataset)
 
         # Need to split new classes dataset indices & old classses dataset indices
-        ann_stats = get_idx_list_per_classes(dataset.dm_subset, True)
+        ann_stats = dataset.get_idx_list_per_classes(True)
```

Comment:
```suggestion
        ann_stats = dataset.get_idx_list_per_classes(use_string_label=True)
```
Pass arguments by keyword when the meaning of the value is not obvious